### PR TITLE
Tweaks mining shed, adds UD relay

### LINKED ISSUE #3572: Mining headsets not going into shortwave mode some shifts
#### Brief description of the issue
For some odd reason, the mining headsets like to sometimes not work at all during rounds. No clue what causes the bug at all besides the fact it's a round long bug when it does occur, so it must be linked to something during initialization, maybe.

#### What you expected to happen
The headsets would work as intended and switch to shortwave when you leave an area with comms.

#### What actually happened
Just pure silence and no shortwave notification at all, sometimes.

#### Steps to reproduce
- None, it just...happens. Needs testing.

#### Code Revision
- release - 
c8a25ab7dd4f89e4af6d7fb736c4e5bf8c211dbc

### PR DESCRIPTION
**Pull request by Arokha**
Fixes #3572 by adding a relay to the Underdark in the mining shed. Also adds a bit of phoron for the generators, and makes it stop raining in the shed.

Also a little #3569 in there

### CODE REVIEW

File: maps/tether/tether_turfs.dm [hunk 1]
```diff
@@ -4,12 +4,14 @@ VIRGO3B_TURF_CREATE(/turf/simulated/open)
...
 	..()
-	outdoor_turfs.Add(src)
+	if(outdoors)
+		outdoor_turfs.Add(src)
 
 VIRGO3B_TURF_CREATE(/turf/simulated/floor)
 /turf/simulated/floor/virgo3b/New()
 	..()
-	outdoor_turfs.Add(src)
+	if(outdoors)
```
Comment (Leshana) [inline maps/tether/tether_turfs.dm]: I think this is redundant with `/turf/simulated/floor/New()` and will double-add.

File: maps/tether/tether_turfs.dm [hunk 2]
```diff
@@ -36,7 +38,8 @@ VIRGO3B_TURF_CREATE(/turf/simulated/floor/reinforced)
 VIRGO3B_TURF_CREATE(/turf/simulated/floor/tiled/steel_dirty)
 /turf/simulated/floor/tiled/steel_dirty/virgo3b/New()
 	..()
-	outdoor_turfs.Add(src)
+	if(outdoors)
```
Comment (Leshana) [inline maps/tether/tether_turfs.dm]: This one is for sure redundant with above New()

File: maps/tether/tether_turfs.dm (hunk 1 above)
Comment (Arokha) [reply maps/tether/tether_turfs.dm]: You are correct. I had no idea that was even a thing. I guess I independently came up with that idea.

File: maps/tether/tether_turfs.dm (hunk 2 above)
Comment (Arokha) [reply maps/tether/tether_turfs.dm]: Though even if I did know, I shouldn't have done this, true.

**Pull request merged**